string command
uint32 point_id
time start_time
time end_time
uint16 red
uint16 white
uint16 led_delay
---
string response
uint32 point_id
time start_time
time end_time
uint16 red
uint16 white
uint16 led_delay